WaysofWork.md

```diff
@@ -297,5 +294,51 @@ it is the time to elevate blockers or important changes to the roadmap or backlo
 As with weeklys and to make use of the synchronous times this meetings have a strong asynchronous component,
 each team will update their stories and make changes to a monthly report PR that the RPM will prepare in advance.
 
+
+#### Planning & Report (meeting outputs)
+
+These issues will be primarly maintained by the RPM and be used to reflect the planned work and progress made either weekly, monthly or quarterly.
+Monthly and quarterly issues will have associated PRs in which the team is expected to take part.
+
+##### Weekly issue
+
+Purpose of this issue is to better communicate and coordinate with DSH team members directly working on project tasks.
+
+It should reflect what was done during the week, blockers and serve to plan work for the following week minimising the need for synchronous meeting time
+
+RPM will be responsible to create and update it each week,
+everyone is responsible to update issues worked on and planned and report in the collaborative note here
+
+The issue will be permanent and updated via comments to avoid drowing the repo, meeting minutes will not be reflected here but stored in Sharepoint and as files in this repo. This issue will therefore only track status of issues so PM can maintian the team's Project.
```

Comment:
```suggestion
The issue will be permanent and updated via comments to avoid drowning the repo.
Meeting minutes will not be reflected here but stored in Sharepoint and as files in this repo. 
This issue will therefore only track status of issues so the PM can maintain the team's Project.
```